telegram_packs/logos.txt

```diff
@@ -0,0 +1,24 @@
+logos/frogAlpine|💿
+logos/frogAndroid|📱
+logos/frogApple|🍏🍎
+logos/frogArch|💿
+logos/frogCemu|🎮
+logos/frogChrome|🌐
+logos/frogDebian|💿
+logos/frogDiscord|✉️📨
+logos/frogDolphinOld|🎮🐬
+logos/frogDolphin|🎮🐬
+logos/frogGentoo|💿
+logos/frogGnome|🖥️
+logos/frogKde|🖥️
```

Comment:
:footprints: and :dragon: for Gnome logo and Konqi respectively?

Reply:
yeah sounds good